src/utils/hotspotUtils.ts

```diff
@@ -1,40 +1,36 @@
 import { round } from 'lodash'
 import { Colors } from '../theme/theme'
 
-export enum SyncStatus {
-  full,
-  partial,
-  none,
-}
-
-export const SYNC_BLOCK_BUFFER = 1500
-
-export const getSyncStatus = (
-  hotspotBlockHeight: number,
-  blockHeight?: number,
-) => {
-  if (!blockHeight)
-    return { status: SyncStatus.none, percent: 0, hotspotBlockHeight }
+export type HotspotSyncStatus = 'full' | 'partial' | 'none'
+
+export const getSyncStatus = ({
+  hotspotBlockHeight,
+  blockHeight,
+  hotspotSyncBuffer,
+}: {
+  hotspotBlockHeight: number
+  blockHeight?: number
+  hotspotSyncBuffer?: number
+}): HotspotSyncStatus => {
+  if (!blockHeight || !hotspotSyncBuffer) {
+    return 'none'
+  }
 
   const syncedRatio = hotspotBlockHeight / blockHeight
   const percentSynced = round(syncedRatio * 100, 2)
```

Comment:
I don't think the concept of percent synced really applies anymore. It's more like hotspot was synced/syncing/offline as of x time ago.

Reply:
I guess maybe diagnostics could use the more traditional sync logic, but maybe that should be separate from the redux/gossip sync logic?